crates/cargo-util-schemas/Cargo.toml

```diff
@@ -9,8 +9,10 @@ repository.workspace = true
 description = "Deserialization schemas for Cargo"
 
 [dependencies]
+schemars = { workspace = true, features = ["preserve_order","semver"], optional = true }
 semver.workspace = true
 serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true, optional = true }
```

Comment:
This will cause `Cargo.lock` to change and that change needs to be included to pass CI